components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
```

Comment:
But then it is added at the end of the line. The idea (maybe wrong) is to ensure that the *FLAGS lines start with references to the needed versions, in case there are other seemingly matching variants later on (e.g. we presumably need specifically zeromq-4.1 and there is 4.2.*-git in /usr/lib and /usr/include)